src/tensorwaves/optimizer/minuit.py

```diff
@@ -69,3 +81,18 @@ def __func(pars: list) -> float:
         results["func_calls"] = function_calls
         results["time"] = end_time - start_time
         return results
+
+
+def dump_parameters(parameters: dict, outputfile: str) -> None:
+    output = {
+        "Parameters": {
+            name: float(value) for name, value in parameters.items()
+        }
+    }
+    with open(outputfile, "w") as yaml_file:
+        yaml.dump(
+            output,
+            yaml_file,
+            sort_keys=False,
+            default_flow_style=False,
+        )
```

Comment:
Just an idea (that feature does not have to also be implemented here): We could think about a feature that appends new Parameters to have really a path of how the Optimizer traveled.

Reply:
That might me usefull, but I dont think that should be enabled by default, since it probably complicates the original task this was implemented for.

Reply:
This could be handled by creating a new `Callback` implementation (#164) and registering values in, say, a `pandas.DataFrame`.

Side note: it would be nice to have the fit running in the background (`asyncio`?), so that you can just `Minuit.optimize` in a notebook and monitor (dynamically -- not by continuously reading back a file) the fit parameters in that `DataFrame`.